packages/buidler-core/src/builtin-tasks/compile.ts

```diff
@@ -235,4 +238,34 @@ export default function () {
     .setAction(async ({ force: force }: { force: boolean }, { run }) =>
       run(TASK_BUILD_ARTIFACTS, { force })
     );
+
+  task(TASK_COMPILE_ADD_COMPILATION_RESULT, async (_, { config }) => {
+    const { default: fetch } = await import("node-fetch");
+
+    try {
+      const compilerVersion = config.solc.version;
+      const solcInputPath = path.join(config.paths.cache, SOLC_INPUT_FILENAME);
+      const solcOutputPath = path.join(
+        config.paths.cache,
+        SOLC_OUTPUT_FILENAME
+      );
+
+      const compilerInput = await fsExtra.readJSON(solcInputPath, {
+        encoding: "utf8",
+      });
+      const compilerOutput = await fsExtra.readJSON(solcOutputPath, {
+        encoding: "utf8",
+      });
+
+      await fetch("http://localhost:8545", {
+        method: "POST",
+        body: JSON.stringify({
+          jsonrpc: "2.0",
+          method: "buidler_addCompilationResult",
+          params: [compilerVersion, compilerInput, compilerOutput],
+          id: 1,
+        }),
+      });
+    } catch (error) {}
```

Comment:
Not sure about what to do here. If there's no node, this will fail, so we shouldn't propagate the error. For the same reason, logging it might be useless.